testing/solidity/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Trevis Schiffer <nikolaibitinit@gmail.com>
+# Maintainer: Trevis Schiffer <nikolaibitinit@gmail.com>
+
+pkgname=solidity
+pkgver=0.5.10
+pkgrel=1
+pkgdesc="The Solidity Contract-Oriented Programming Language"
+url="https://github.com/ethereum/solidity"
+arch="all"
+license="GPL3"
+depends=""
+makedepends="boost-dev cmake git"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/ethereum/$pkgname/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+options="!check" # Test suite issues
+
+prepare() { 
```

Comment:
missing default_prepare on top

Reply:
Reference says it only requires for applying patches..
What I am missing in this case?

Reply:
It is expected to leave there so people aren't surprised why their patches aren't applied

Reply:
I see